### PR TITLE
New package: zpaq

### PR DESCRIPTION
**Pull request by necrophcodr**
This commit adds the package "zpaq", for incremental archiving purposes.
The package installs a single binary `zpaq`into /usr/bin.
For more information on this application, please consult the homepage of this package ( http://mattmahoney.net/dc/zpaq.html )

### CODE REVIEW

File: srcpkgs/zpaq/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="Incremental Journaling Backup Utility and Archiver"
+build_style=gnu-makefile
+maintainer="necrophcodr <necrophcodr@necrophcodr.me>"
+homepage="http://mattmahoney.net/dc/zpaq.html"
+distfiles="http://mattmahoney.net/dc/zpaq660.zip"
+wrksrc="."
+checksum=58fe7d94421ac115fb04eb58bacc17093ebd1294b05320989bcb1ce3be29cbdc
+
+do_install() {
+	vinstall zpaq 0755 usr/bin
```
Comment (dominikh) [inline srcpkgs/zpaq/template]: Use `vbin`

**Comment by xtraeme** [comment]
wrksrc is completely wrong.

**Comment by xtraeme** [comment]
`unzip` must be added to `hostmakedepends`

**Comment by necrophcodr** [comment]
wrksrc is not completely wrong. it extracts to the current directory. should it be different somehow?

**Comment by xtraeme** [comment]
`wrksrc="."` is completely wrong because that means that `xbps-src clean` won't cleanup `$XBPS_BUILDDIR` correctly. Use `create_wrksrc=yes` to create a default wrksrc.

**Comment by xtraeme** [comment]
Isn't there a manpage or a README?

**Comment by necrophcodr** [comment]
Yeah, it does read that on line 341, I was just not aware to use it under these circumstances. Has been fixed. No need to be an ass.

**Comment by xtraeme** [comment]
I wasn't being an 'ass' and you are not following the commit rules, I'm now an 'ass'.

Please resubmit a proper pull request as explained in CONTRIBUTING.md.

Thanks.

**Comment by xtraeme** [comment]
Please submit a proper commit, force-pushing to that PR.

**Comment by dominikh** [comment]
The following

> Isn't there a manpage or a README?

wasn't a snarky way of saying "RTFM", it was asking if zpaq didn't come with any sort of manpage or README.

**Comment by necrophcodr** [comment]
It does now!

**Comment by xtraeme** [comment]
Looks good now, thanks :+1:

**Pull request merged**